modules/veteran_confirmation/VETERAN_CONFIRMATION.yml

```diff
@@ -29,13 +29,13 @@ info:
     ## Authorization
     This API requires an API key in combination with identifiable information for the person being confirmed listed below. API requests are authorized through a symmetric API token provided in an HTTP header with name `apikey`. Including more information has a better chance of making a match and returning a Confirmed status. 
     #### Required information:
-    * First Name  
-    * Last Name  
-    * Date of Birth  
+    * First Name 
+    * Last Name 
+    * Date of Birth 
```

Comment:
Any idea what git is up to here?

Reply:
I see an extra symbol at the end of these lines on master, probably leftovers from copy/pasting from Figma. Thanks for taking care of those @rtravitz!

Reply:
Removed some hidden characters